data_model/src/trigger.rs

```diff
@@ -64,62 +64,38 @@ pub mod model {
     )]
     #[display(fmt = "@@{id}")]
     #[ffi_type]
-    pub struct Trigger<F, E> {
+    pub struct Trigger<F> {
         /// [`Id`] of the [`Trigger`].
         pub id: TriggerId,
```

Comment:
can you go to `TriggerId` definition and swap `name` and `domain_id`. So that `Ord` impl orders first by domain then by name